apps/desktop/src/lib/commit/StackingUpstreamCommitsAccordion.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+	import Icon from '@gitbutler/ui/Icon.svelte';
+	import type { Snippet } from 'svelte';
+
+	interface Props {
+		children: Snippet;
+		action: Snippet;
+		remoteCommitCount: number;
+	}
+
+	let isOpen = $state(false);
+
+	const { children, action, remoteCommitCount }: Props = $props();
```

Comment:
Just for good form, let's move this right underneath the `Props` interface declaration.